docs/reference/search/request/highlighting.asciidoc

```diff
@@ -200,8 +208,14 @@ handy when you need to highlight short texts such as a title or
 address, but fragmentation is not required. If `number_of_fragments`
 is 0, `fragment_size` is ignored. Defaults to 5.
 
-order:: Sorts highlighted fragments by score when set to `score`. Only valid for
-the `unified` highlighter.
+order:: Sorts highlighted fragments by score when set to `score`.  By default,
+fragments will be output in the order they appear in the field (order: `none`).
+Setting this option to `score` will output the most relevant fragments first.
+Only `unified` highlighter truly calculates the score in a similar way the score
```

Comment:
Each highlighter has its own relevancy "sauce" but I wouldn't say that the `unified` scoring is similar to the score of the query. It uses BM25 but that's just an internal detail, I think we should just say that each highlighter applies its own logic to compute the relevancy score and we can describe the details in the example below.